src/components/Timeseries/index.tsx

```diff
@@ -24,6 +24,8 @@ import {
     setPopup,
 } from 'redux/App/slice';
 import { getCountryDataFromTimeseriesData } from 'utils/helperFunctions';
+import { URLToFilters } from '../../utils/helperFunctions';
```

Comment:
In this application we have `absolute paths` set up so you don't have to reference files using relative paths. You can change `../../utils...` to just `utils/helperFunctions`